mitmproxy/addons/share.py

```diff
@@ -0,0 +1,73 @@
+import typing
+import random
+import datetime
+import time
+import _io
```

Comment:
This is a really bad idea, we definitely want to import `io` itself here. You can replace `from mitmproxy import io` with `import mitmproxy.io`.

Reply:
I didn't know that `_io` is the internal code that `io` uses. I will change it